cloudformation/jolly-roger.yaml

```diff
@@ -650,25 +649,32 @@ Resources:
                           </script>
                         </body>
                       </html>
+                  - path: /etc/haproxy/haproxy.cfg
+                    content: |
+                      global
+                      maxconn 50000
+                      log stdout local0
+
+                      defaults
+                      log global
+                      timeout connect 5s
+                      timeout client 50s
+                      timeout server 50s
+
+                      frontend inbound
+                      bind :443
+                      tcp-request inspect-delay 5s
+                      acl is_turn payload(0,2) -m bin 0000
```

Comment:
Bah. This is actually incorrect. The prefix is defined as [part of the STUN protocol](https://datatracker.ietf.org/doc/html/rfc8489#section-5) (TURN adds additional methods to the STUN protocol, rather than being its own protocol), but it's a two-*bit* prefix, not a two-byte prefix.

That's actually enough to distinguish it from HTTP (the first byte of an HTTP/1 or HTTP/2 request is always A-Z, which means that the 2nd most significant bit is set), but two bits feels more fragile than two bytes. I'll adjust to pick up the magic cookie, which starts at byte 4

Reply:
(You did interpret it correctly, though)

Reply:
For what it's worth, this config as written actually work at all, and I was just misinterpreting the results from the testing tool I was using, but I now have a config that works. (I'll push it up once I finish fixing the nginx config)